getting-started-tutorial/tutorial-2-Model-training.ipynb

```diff
@@ -0,0 +1,2021 @@
+{
```

Comment:
<p>Why call the <code>code_to_function</code>  <code>gen_func</code>  when it's actually the <code>train_iris</code>  function (and then actually getting the setted function from the project as <code>train_iris_func</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>